app/models/amusement_park.rb

```diff
@@ -1,3 +1,13 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
+  has_many :mechanic_rides, through: :rides
+  has_many :mechanics, through: :mechanic_rides
```

Comment:
Great work adding these additional associations! In the future, you can skip right to mechanics like so:
`has_many :mechanics, through: :rides`